Source/Scene/GroundPrimitive.js

```diff
@@ -807,6 +903,19 @@ define([
         }
     }
 
+    GroundPrimitive._initialized = false;
+    GroundPrimitive.init = function() {
+        var initialized = GroundPrimitive._initialized;
+        if (initialized) {
+            return when.resolve();
```

Comment:
You should store and return the promise below, rather than returning a new promise every time here.